sources/HUBCollectionViewLayout.m

```diff
@@ -183,7 +183,7 @@ - (CGPoint)targetContentOffsetForProposedContentOffset:(CGPoint)proposedContentO
     
     for (NSIndexPath *indexPath in self.lastViewModelDiff.insertedBodyComponentIndexPaths) {
         if (indexPath.item <= topmostVisibleIndex) {
-            UICollectionViewLayoutAttributes *attributes = self.previousLayoutAttributesByIndexPath[indexPath];
+            UICollectionViewLayoutAttributes *attributes = self.layoutAttributesByIndexPath[indexPath];
```

Comment:
The size of the added cells should be computed from the new layout attributes, rather than the previous.